config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -288,6 +288,32 @@ public ServerHttpSecurity addFilterAt(WebFilter webFilter, SecurityWebFiltersOrd
 		return this;
 	}
 
+	/**
+	 * @since 5.2.0
+	 * @author Rob Winch
```

Comment:
Please move `@since` and `@author` to the bottom of the Javadoc (just below `@return`)

Reply:
Moved @since and @author after @since after at return.
And author name should we one who reviews pull request?
And please help with tets. No idea on same.

Reply:
Added test please review